### DISCUSSION TITLE
Value is not registering in custom react select component

### DISCUSSION THREAD

**Discussion by azad-steadybiz** [react-hook-form/react-hook-form#2188] [Q&A] [answered]
Value is not registering in custom react select component, getting validation error all the time after selecting value also.

sandbox: https://codesandbox.io/s/strange-knuth-mkxkg

**Comment by azad-steadybiz** [comment, marked as answer]
@bluebill1049  Bro, please help

**Reply by bluebill1049** [reply]
hey @azad-steadybiz please send some time with the doc, read the `render` part, I can jump in and help with the code, but you should value my time as well so i can prioritize other propblems, I can't just keep jump in and investigate the code problem. Here is the working version with `render` prop: https://codesandbox.io/s/frosty-star-17z3c?file=/src/widgets/MultiSelect/index.js

**Reply by azad-steadybiz** [reply]
Sorry for the trouble, but it was perfectly fine. but after some changes I don't know what went wrong and it stopped working. Still I can't get it.

**Reply by bluebill1049** [reply]
can you see my codesandbox? it's working for me here.

**Reply by bluebill1049** [reply]
<img width="1205" alt="Screen Shot 2020-07-13 at 10 06 55 pm" src="https://user-images.githubusercontent.com/10513364/87302671-2ca31a00-c555-11ea-911e-081015e45c83.png">

Take a look at the screenshot above.
